packages/wifi_info_flutter/wifi_info_flutter/example/lib/main.dart

```diff
@@ -103,7 +103,7 @@ class _MyHomePageState extends State<MyHomePage> {
   Future<void> _updateConnectionStatus(ConnectivityResult result) async {
     switch (result) {
       case ConnectivityResult.wifi:
-        String wifiName, wifiBSSID, wifiIP;
+        String? wifiName, wifiBSSID, wifiIP;
```

Comment:
Are these actually `?`? They look `late`.

Reply:
Yeah, the getter methods returning these infos returns nullables:
For example in `wifiBSSID = await _wifiInfo.getWifiBSSID()`, _wifiInfo.getWifiBSSID() returns `Future<String?>`